src/Core/Checkout/Customer/Rule/BillingCityRule.php

```diff
@@ -57,7 +57,7 @@ public function getConstraints(): array
         $constraints = [
             'operator' => [
                 new NotBlank(),
-                new Choice([Rule::OPERATOR_EQ, Rule::OPERATOR_NEQ, Rule::OPERATOR_EMPTY]),
+                new Choice(choices: [Rule::OPERATOR_EQ, Rule::OPERATOR_NEQ, Rule::OPERATOR_EMPTY]),
```

Comment:
can we increase composer.json package to the min version when this was introduced?